### PR TITLE
Chapters

### PR DESCRIPTION
**Pull request by ntokozo-nhlangulela**

### CODE REVIEW

File: src/agent/api.ts
Comment (CharelMarais) [inline src/agent/api.ts:1]: Remove any unused commented code

File: src/components/mangaCard.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import Typography from "@mui/material/Typography";
+import { useEffect, useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import MangaApis from "../agent/api";
+import MangaList from "../models/mangaList";
+import appPaths from "../shared/paths";
+
+//coverArt: item.relationships.attributes.fileName,
+
+const T = () => {
```
Comment (CharelMarais) [inline src/components/mangaCard.tsx:16]: What is T what does it do?

File: src/components/mangaCard.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+              fileName: name,
+            },
+          ]);
+        });
+      });
+    }
+  }, []);
+
+  const getMangoDetails = (mangaId: string) => {
+    // MangaApis.getMangaDetails(mangaId).then((response) => {
```
Comment (CharelMarais) [inline src/components/mangaCard.tsx]: Commented code to be removed

File: src/components/mangaDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+import { useEffect, useRef, useState } from "react";
+import MangaApis from "../agent/api";
+import MangaList from "../models/mangaList";
+
+interface mangaDetailsProps {
+  mangaInformation: MangaList;
+}
+
+const MangaDetails = () => {
+  //const { mangaInformation } = props;
```
Comment (CharelMarais) [inline src/components/mangaDetails.tsx]: Commented Code

File: src/components/mangaDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  //ToDo: Refactor this useEffect
+  useEffect(() => {
+    if (!isRendered.current) {
+      isRendered.current = true;
+      MangaApis.getAllMangas().then((response) => {
+        const matchingRecord = response.data.find(
+          (item: any) => item.id === mangaId
+        );
+
+        let name = "";
```
Comment (CharelMarais) [inline src/components/mangaDetails.tsx:49]: You need to set the types.

**Review by CharelMarais — COMMENTED** [review]
Please check the comments and make adjustments where needed. You can merge after that

File: src/components/mangaCard.tsx (hunk 1 above)
Comment (ntokozo-nhlangulela) [reply src/components/mangaCard.tsx:16]: I still need to refactor my code, this will include renaming my components.

**Review by KiveshVirasamy — APPROVED** [review]
Looks good

**Pull request merged**